system/Makefile.am

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2015 Alon Bar-Lev <alon.barlev@gmail.com>
+#
+MAINTAINERCLEANFILES= $(srcdir)/Makefile.in
+CLEANFILES = \
+	netdata.service \
+	$(NULL)
+
+include $(top_srcdir)/build/subst.inc
+
+SUFFIXES = .in
+
+dist_noinst_DATA = \
```

Comment:
The difference here is we want to package the `netdata.service.in`, not the `netdata.service` I think.